### ISSUE TITLE
3D (synthetic) preprocessing

### ISSUE THREAD

**Issue by sebapusch** [sebapusch/X2CT-MMe3D#12]
Implement preprocessing pipeline for 3D synthetic CT scan data. The output will be used as input to PyTorch models and must follow conventions for `torch.utils.data.Dataset`

Each volume must be processed as follows:

- [x] Resample to consistent voxel spacing (e.g., [1.0, 1.0, 1.0] mm)
- [x] Define intensity range (e.g. [-1000, 1000])
- [x] Clip pixel values accordingly
- [x] Pixel normalization
- [x] Normalize intensities to [0, 1] or standardize (mean/std)
- [x] Document method used
- [x] Convert to PyTorch tensors,  Shape: (C, D, H, W), Type: torch.float32
